grpc/server.proto

```diff
@@ -14,6 +14,11 @@ message SubscribeRequest {
   string topic = 1;
   string subscription_id = 2;
   map<string, string> filter = 3;
+  int64 retention_period = 4;
+  int64 visibility_timeout = 5;
```

Comment:
we might want to go with [UInt64Value types](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/u-int64-value) here, since they're optional